extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderLightbox.php

```diff
@@ -81,36 +82,46 @@
 					</select>
 				</div>
 				<div class="input-group">
-					<label for="occupation"><?= wfMsg('user-identity-box-about-occupation'); ?></label>
-					<input type="text" name="occupation" value="<?= $user['occupation']; ?>" maxlength="<?= $charLimits['occupation']; ?>" />
+					<label for="occupation"><?= wfMessage( 'user-identity-box-about-occupation' )->escaped(); ?></label>
+					<input type="text" id="occupation" name="occupation" value="<?= $user['occupation']; ?>" maxlength="<?= $charLimits['occupation']; ?>"/>
+				</div>
+				<div class="input-group">
+					<label for="gender"><?= wfMessage( 'user-identity-box-about-gender' )->escaped(); ?></label>
+					<input type="text" id="gender" name="gender" value="<?= $user['gender']; ?>" maxlength="<?= $charLimits['gender']; ?>"/>
 				</div>
 				<div class="input-group">
-					<label for="gender"><?= wfMsg('user-identity-box-about-gender'); ?></label>
-					<input type="text" name="gender" value="<?= $user['gender']; ?>" maxlength="<?= $charLimits['gender']; ?>" />
+					<label for="website"><?= wfMessage( 'user-identity-box-about-website' )->escaped(); ?></label>
+					<input type="text" id="website" name="website" value="<?= $user['website']; ?>">
 				</div>
 				<div class="input-group">
-					<label for="website"><?= wfMsg('user-identity-box-about-website'); ?></label>
-					<input type="text" name="website" value="<?= $user['website']; ?>">
+					<label for="twitter"><?= wfMessage( 'user-identity-box-about-tweet' )->escaped(); ?></label>
+					<label for="twitter" class="tweet-at">twitter.com/</label>
+					<input type="text" id="twitter" name="twitter" value="<?= $user['twitter']; ?>">
 				</div>
 				<div class="input-group">
-					<label for="twitter"><?= wfMsg('user-identity-box-about-tweet'); ?></label>
-					<span class="tweet-at">@</span>
-					<input type="text" name="twitter" value="<?= $user['twitter']; ?>">
+					<label for="fbPage"><?= wfMessage( 'user-identity-box-about-fb-page' )->escaped(); ?></label>
+					<label for="fbPage" class="fb-start">facebook.com/</label>
+					<input type="text" id="fbPage" name="fbPage" value="<?= $user['fbPage']; ?>">
```

Comment:
Escaping single quote sounds ok for links, on the other hand this methos is escaping other attributes like ```realName``` which imo, should be able to display single quotes.
@jsutterfield : What do you think?